src/cascadia/TerminalSettingsEditor/KeyChordListener.cpp

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "KeyChordListener.h"
+#include "KeyChordListener.g.cpp"
+#include "LibraryResources.h"
+
+using namespace winrt::Windows::UI::Core;
+using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Xaml::Controls;
+using namespace winrt::Windows::UI::Xaml::Data;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::System;
+using namespace winrt::Windows::UI::Xaml::Input;
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    DependencyProperty KeyChordListener::_KeysProperty{ nullptr };
+
+    static constexpr std::array<VirtualKey, 10> ModifierKeys{
```

Comment:
Just `std::array` is enough.

Also: Do you mind sorting these members by their numerical value and document this as such?
I doubt we'll invent new modifier keys anytime soon, so I think it's worth the effort to nail the list perfectly the first time. 😄

Reply:
Oh right: This is because if the list is sorted, the CPU will love it's predictability. But more importantly we can use binary search on it that way.